flexmeasures/data/migrations/versions/994170c26bc6_add_account_table.py

```diff
@@ -111,26 +111,30 @@ def upgrade_data():
             print(
                 f"Linking user {user_id} to account {account_name} (as from custom param) ..."
             )
-            account_results = (
-                session.query(Account.id).filter_by(name=account_name).one_or_none()
-            )
+            account_results = session.execute(
+                sa.select(Account.id).filter_by(name=account_name)
+            ).scalar_one_or_none()
             if account_results is None:
                 print(f"need to create account {account_name} ...")
                 account = Account(name=account_name)
                 session.add(account)
                 session.flush()
                 account_id = account.id
             else:
-                account_id = account_results[0]
-            user_results = session.query(User.id).filter_by(id=user_id).one_or_none()
+                account_id = account_results
+            user_results = session.execute(
+                sa.select(User.id).filter_by(id=user_id)
+            ).scalar_one_or_none()
             if not user_results:
                 raise ValueError(f"User with ID {user_id} does not exist!")
             connection.execute(
                 f"UPDATE fm_user SET account_id = {account_id} WHERE id = {user_id}"
             )
 
     # Make sure each existing user has an account
-    for user_results in session.query(User.id, User.email, User.account_id).all():
+    for user_results in session.execute(
+        sa.select(User.id, User.email, User.account_id)
+    ).all():
```

Comment:
If I use `scalars` here, I would only get the `id` and not the other two columns data. So that's why I used `execute`. I can show you this in an example.

Reply:
Okay, got it. Glad to learn this.